examol/start/kmeans.py

```diff
@@ -1,16 +1,42 @@
 """Algorithms which select molecules based on k-means clustering"""
+import numpy as np
+from sklearn.cluster import KMeans
 from .base import Starter
 from examol.score.rdkit.descriptors import compute_morgan_fingerprints
+from scipy.spatial.distance import cdist
 
 
 class KMeansStarter(Starter):
     """Select structurally distinct molecules by picking molecules at the centers of clusters"""
 
     def _select(self, to_select: list[str], count: int) -> list[str]:
-        # Compute the fingerprints for each molecule
-        finger = [
-            compute_morgan_fingerprints(x) for x in to_select
-        ]
-
-        # Pick the molecules at the center of each cluster
-        raise NotImplementedError()
+        """
+        Input:
```

Comment:
Most Python IDEs will copy the documentation from the superclass, so let's delete the copy here to avoid needing to keep both up-to-date with any wording changes.